src/controllers/reporting.controller.ts

```diff
@@ -18,7 +18,23 @@ export const getZipAlertList = asyncHandler(
         req: Request<{}, {}, {}, {}>,
         res: Response,
         next: NextFunction
-    ) => {}
+    ) => {
+        //var arr: Number[] = []
+        var { zipList } = require('../config/queue.ts');
+        if(zipList != null){
```

Comment:
Is it possible for the list to be null? If not, I don't think we need this check.

Reply:
Yeah it's probably unnecessary. It never returned as null when I was testing it, but the statewide alert controller threw a fit when I tried to pass in the alertMap, so I just wanted to be safe.